spring-rabbit/src/test/java/org/springframework/amqp/rabbit/connection/SSLConnectionTests.java

```diff
@@ -94,8 +94,6 @@ public void testKSTS() throws Exception {
 		Log logger = spy(TestUtils.getPropertyValue(fb, "logger", Log.class));
 		given(logger.isDebugEnabled()).willReturn(true);
 		new DirectFieldAccessor(fb).setPropertyValue("logger", logger);
-		ConnectionFactory rabbitCf = spy(TestUtils.getPropertyValue(fb, "connectionFactory", ConnectionFactory.class));
-		new DirectFieldAccessor(fb).setPropertyValue("connectionFactory", rabbitCf);
```

Comment:
The rest new tests have the same redundant code.

Will fix on merge after answer to `Arrays.toString()` question.